Makefile

```diff
@@ -1,63 +1,139 @@
+#
+# Makefile for the Links Programming Language.
+#
+
+# This file contains instructions for building a bleeding-edge version
+# of Links in release or development mode with and without database
+# support.
+
+# LATEST STABLE RELEASE
+# =====================
+# To install the latest stable release of Links please use opam, e.g.
+# $ opam install links
+# .. and to enable database support
+# $ opam install links-postgresql
+
+# To build the bleeding-edge version of Links please refer to the
+# instructions below.
+
+# DEVELOPMENT
+# ===========
+# By default the build rule `all' attempts to build the latest
+# development version of Links with database support, thus the
+# following command suffices to build everything
+# $ make all
```

Comment:
If `all` is the default target why not just instruct users to run `make` instead of `make all`?

Reply:
I prefer to be explicit here. The intended audience is you, me, and the rest of the Links developers. However, I did leave a note about how to install the latest stable version in case some bewildered user happens to look inside this file.